### ISSUE TITLE
Feature: Allow headless/nogui to be a global setting

### ISSUE THREAD

**Issue by nigelkersten** [thbishop/fission#10]
I'd find it really useful to be able to declare in my .fissionrc that I want all actions to be headless by default.

**Comment by thbishop** [comment]
Yep, this makes sense.  I think support for overriding of the global setting in an ad hoc fashion should be also be included.
